.github/workflows/ci_tests.yml

```diff
@@ -20,23 +20,23 @@ jobs:
     name: Mandatory checks before CI
     runs-on: ubuntu-latest
     steps:
-    - name: Check base branch
-      uses: actions/github-script@v3
-      if: github.event_name == 'pull_request'
-      with:
-        script: |
-          const skip_label = 'skip-basebranch-check';
-          const { default_branch: allowed_basebranch } = context.payload.repository;
-          const pr = context.payload.pull_request;
-          if (pr.labels.find(lbl => lbl.name === skip_label)) {
-            core.info(`Base branch check is skipped due to the presence of ${skip_label} label`);
-            return;
-          }
-          if (pr.base.ref !== allowed_basebranch) {
-            core.setFailed(`PR opened against ${pr.base.ref}, not ${allowed_basebranch}`);
-          } else {
-            core.info(`PR opened correctly against ${allowed_basebranch}`);
-          }
+      - name: Check base branch
```

Comment:
Is this from a linter?

Reply:
If so, then LGTM. If not, then let's have a CI linter and never discuss this again 😆

Reply:
Yeah this is coming from prettier AFAIK